media-sound/helm/helm-9999.ebuild

```diff
@@ -0,0 +1,58 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit eutils
+
+DESCRIPTION="A free polyphonic synth with lots of modulation"
+HOMEPAGE="http://tytel.org/helm/"
+LICENSE="GPL-3+ CC-BY-3.0"
+if [[ ${PV} == *9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/mtytel/helm.git"
+	KEYWORDS=""
+else
+	SRC_URI="https://github.com/mtytel/helm/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64"
+fi
+SLOT="0"
+
+IUSE="standalone lv2 vst"
+REQUIRED_USE="|| ( standalone lv2 vst )"
```

Comment:
Yeah, since the VST3 SDK was made available under a usable license some projects have included it in their codebase. Helm does that as well, see https://github.com/mtytel/helm/tree/master/VST3_SDK